packages/console/src/components/CreateTenantModal/SelectTenantPlanModal/SkuCardItem/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { pricingLink } from '@/consts';
+import { TenantsContext } from '@/contexts/TenantsProvider';
+import Button from '@/ds-components/Button';
+import DangerousRaw from '@/ds-components/DangerousRaw';
+import DynamicT from '@/ds-components/DynamicT';
+import TextLink from '@/ds-components/TextLink';
+
+import * as styles from '../PlanCardItem/index.module.scss';
+
+import FeaturedSkuContent from './FeaturedSkuContent';
+
+type Props = {
+  readonly sku: LogtoSkuResponse;
+  readonly onSelect: () => void;
+  readonly buttonProps?: Partial<React.ComponentProps<typeof Button>>;
+};
+
+function SkuCardItem({ sku, onSelect, buttonProps }: Props) {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console.upsell.create_tenant' });
+  const { tenants } = useContext(TenantsContext);
+  const { unitPrice: basePrice, id: skuId } = sku;
+
+  const isFreeSku = skuId === ReservedPlanId.Free;
+
+  const isFreeTenantExceeded = useMemo(
+    () =>
+      /** Should not block admin tenant owners from creating more than three tenants */
+      !tenants.some(({ id }) => id === adminTenantId) &&
+      tenants.filter(({ planId }) => planId === ReservedPlanId.Free).length >= maxFreeTenantLimit,
+    [tenants]
+  );
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.planInfo}>
+        <div className={styles.title}>
+          <PlanName skuId={skuId} name={skuId} />
+        </div>
+        <div className={styles.priceInfo}>
+          <div className={styles.priceLabel}>{t('base_price')}</div>
+          <div className={styles.price}>
+            ${t('monthly_price', { value: (basePrice ?? 0) / 100 })}
+          </div>
+        </div>
+        <div className={styles.description}>
+          <PlanDescription skuId={skuId} planId={skuId} />
+        </div>
+      </div>
+      <div className={styles.content}>
+        <FeaturedSkuContent skuId={skuId} />
+        {isFreeSku && isFreeTenantExceeded && (
+          <div className={classNames(styles.tip, styles.exceedFreeTenantsTip)}>
+            {t('free_tenants_limit', { count: maxFreeTenantLimit })}
+          </div>
+        )}
+        {!isFreeSku && (
+          <div className={styles.tip}>
+            <TextLink
+              isTrailingIcon
+              href={pricingLink}
+              targetBlank="noopener"
+              icon={<ArrowRight className={styles.linkIcon} />}
+              className={styles.link}
+            >
+              <DynamicT forKey="upsell.create_tenant.view_all_features" />
+            </TextLink>
+          </div>
+        )}
+        <Button
+          title={
+            <DangerousRaw>
+              <Trans components={{ name: <PlanName skuId={skuId} name={skuId} /> }}>
+                {t('select_plan')}
+              </Trans>
+            </DangerousRaw>
+          }
+          type={isFreeSku ? 'outline' : 'primary'}
+          size="large"
+          onClick={onSelect}
+          {...buttonProps}
```

Comment:
Should we put this at the very first beginning? Otherwise, the properties you defined above (title, type, size, and onClick) would probably be overriden by the `buttonProps` passed from outside.

Reply:
Or maybe this is on purpose?

Reply:
ok